wails-frontend/src/pages/PasswordPromptHandler/Sign.tsx

```diff
@@ -105,14 +119,47 @@ export function Sign() {
           {(() => {
             switch (signData.OperationType) {
               case OPER_CALL_SC:
-                return <CallSc {...signData} />;
+                return (
```

Comment:
Concerning the fees being disabled during, i believe that passing isEditing as a param to OperationCost would make sense... tdb in small follow up pr